src/host/getset.cpp

```diff
@@ -1356,24 +1356,21 @@ void DoSrvPrivateAllowCursorBlinking(SCREEN_INFORMATION& screenInfo, const bool
         if (screenInfo.IsCursorInMargins(oldCursorPosition))
         {
             // Cursor is at the top of the viewport
-            const COORD bufferSize = screenInfo.GetBufferSize().Dimensions();
             // Rectangle to cut out of the existing buffer
             SMALL_RECT srScroll;
             srScroll.Left = 0;
-            srScroll.Right = bufferSize.X;
+            srScroll.Right = SHORT_MAX;

```

Comment:
Please leave comment on why SHORT_MAX is acceptable.

Reply:
Done.